platform/ios/app/MBXViewController.m

```diff
@@ -167,153 +216,263 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(__unused id)sender {
-        @"Add Test Shapes",
-        @"Start World Tour",
-        @"Add Custom Callout Point",
-        @"Remove Annotations",
-        @"Manipulate Style",
-        ((_customUserLocationAnnnotationEnabled)
-         ? @"Disable Custom User Dot"
-         : @"Enable Custom User Dot"),
-        nil];
-
-    if (self.debugLoggingEnabled)
+
+    switch (section)
     {
-        [sheet addButtonWithTitle:@"Print Telemetry Logfile"];
-        [sheet addButtonWithTitle:@"Delete Telemetry Logfile"];
+        case MBXSettingsCoreRendering:
+            [settingsTitles addObjectsFromArray:@[
+                @"Reset Position",
+                [NSString stringWithFormat:@"%@ Tile Boundaries",
+                    (debugMask & MGLMapDebugTileBoundariesMask ? @"Hide" :@"Show")],
+                [NSString stringWithFormat:@"%@ Tile Info",
+                    (debugMask & MGLMapDebugTileInfoMask ? @"Hide" :@"Show")],
+                [NSString stringWithFormat:@"%@ Tile Timestamps",
+                    (debugMask & MGLMapDebugTimestampsMask ? @"Hide" :@"Show")],
+                [NSString stringWithFormat:@"%@ Collision Boxes",
+                    (debugMask & MGLMapDebugCollisionBoxesMask ? @"Hide" :@"Show")],
+                [NSString stringWithFormat:@"%@ Overdraw Visualization",
+                    (debugMask & MGLMapDebugOverdrawVisualizationMask ? @"Hide" :@"Show")],
+            ]];
+            break;
+        case MBXSettingsAnnotations:
+            [settingsTitles addObjectsFromArray:@[
+                @"Add 100 Views",
+                @"Add 1,000 Views",
+                @"Add 10,000 Views",
+                @"Add 100 Sprites",
+                @"Add 1,000 Sprites",
+                @"Add 10,000 Sprites",
+                @"Add Test Shapes",
+                @"Add Point With Custom Callout",
+                @"Remove Annotations",
+            ]];
+            break;
+        case MBXSettingsRuntimeStyling:
+            [settingsTitles addObjectsFromArray:@[
+                @"Apply Water Functions",
+                @"Apply Road Line Functions",
+                @"Create Raster & Apply Function",
+                @"Create GeoJSON & Apply Fill",
+                @"Apply Symbol Color",
+                @"Apply Building Fill Color",
+                @"Apply Ferry Line Color",
+                @"Remove Park Layer",
+            ]];
+            break;
+        case MBXSettingsMiscellaneous:
+            [settingsTitles addObjectsFromArray:@[
+                @"Start World Tour",
+                [NSString stringWithFormat:@"%@ Custom User Dot", (_customUserLocationAnnnotationEnabled ? @"Disable" : @"Enable")],
+            ]];
+
+            if (self.debugLoggingEnabled)
+            {
+                [settingsTitles addObjectsFromArray:@[
+                    @"Print Telemetry Logfile",
+                    @"Delete Telemetry Logfile",
+                ]];
+            };
+            break;
+        default:
+            break;
```

Comment:
Should be this an assertion, in case a contributor forgets to add a case here or adds it to the wrong switch statement?